File: CarRent/src/app/shared/models/order.model.ts

import { User } from "./user-info.model";
import { Car } from "./car-info.model";

export interface Order {
    OrderId?:number;
    StartRentDate: Date;
    ReturnDate?: Date;
    EndOfRentDate: Date;
    UserId?:number;
    CarId?:number;
    Car?:Car;
    User?:User;
    TotalPrice?:number;
}

File: CarRentProject/BOL/OrderModel.cs
using System;


namespace BOL
{
    public class OrderModel
    {
        public int OrderId { get; set; }
        public DateTime StartRentDate { get; set; }
        public DateTime? ReturnDate { get; set; }
        public DateTime EndOfRentDate { get; set; }
        public int UserId { get; set; }
        public int CarId { get; set; }
        public UserModel User { get; set; }
        public CarModel Car { get; set; }

    }
}
